data-prepper-api/src/test/java/org/opensearch/dataprepper/model/event/JacksonEventTest.java

```diff
@@ -567,8 +602,9 @@ public void testBuild_withFormatStringWithExpressionEvaluator() {
 
         final ExpressionEvaluator expressionEvaluator = mock(ExpressionEvaluator.class);
 
-        when(expressionEvaluator.isValidExpressionStatement("foo")).thenReturn(false);
+        verify(expressionEvaluator, times(0)).isValidExpressionStatement(eventKey);
```

Comment:
There is also a `never()` you can use instead of `times(0)`. It's a bit more readable, but should have the same result. Also, I'd probably put an `anyString()` in there instead of `eventKey`. When verifying that something is not called, I prefer to be as loose as possible.